src/apps/backend/documentation/index.ts

```diff
@@ -0,0 +1,13 @@
+import App from '../app';
+
+import expressListRoutes from './express-list-routes';
+
+(() => {
+  const servers = App.getAllServers();
```

Comment:
You want to still DocumentationService and do the logic there and have it call here.